### ISSUE TITLE
Add bulk download CSV water level requests

### ISSUE THREAD

**Issue by kinverarity1** [kinverarity1/python-sa-gwdata#1]
They don't have all the data but they're a damn sight quicker than doing individual requests.

e.g. https://www.waterconnect.sa.gov.au/_layouts/15/dfw.sharepoint.wdd/WDDDMS.ashx/GetWaterLevelDownload?bulkOutput=CSV

In the POST content the key `exportdata` appears to have string-encoded JSON:

```
{"Box":+[-35.606559,139.234534,-35.158702,140.460883],
+"DHNOs":+[83454,83495,83498,83499,83502,83504,83516,83527,83577,
83644,83646,83647,83654,83663,83664,83665,83666,83703,84741,84751,
84791,84792,84793,84795,84802,84803,84868,84878,84879,148693,167427,
171071,173662,182486,182487,190264,192545,196037,196355,200067,
201427,201465,140892,212313,232824,234615,238843,240307,102041,
205988],+"Type":+"CSV",+"Anomalous":+true,+"Pumping":+true}
```

August update:

The full list of bulk data service names:

- [x] GetSummaryDownload
- [x] GetWaterLevelDownload
- [x] GetSalinityDownload
- [x] GetWaterChemistryDownloadAllData
- [x] GetConstructionDownload
- [x] GetConstructionDetailsDownload (returns ZIP)
- [x] GetDrillersLogDownload
- [x] GetElevationDownload
- [x] GetLithologicalLogDownload
- [x] GetHydroStratLogDownload
- [x] GetStratLogDownload

**Comment by kinverarity1** [comment]
The full list of bulk data service names:

- GetSummaryDownload
- GetWaterLevelDownload
- GetSalinityDownload
- GetWaterChemistryDownloadAllData
- GetConstructionDownload
- GetConstructionDetailsDownload (returns ZIP)
- GetDrillersLogDownload
- GetElevationDownload
- GetLithologicalLogDownload
- GetHydroStratLogDownload
- GetStratLogDownload

**Comment by kinverarity1** [comment]
Completed and released in v0.10.0
